app/src/features/surveys/view/survey-animals/TelemetryDeviceForm.tsx

```diff
@@ -22,48 +29,95 @@ export interface IAnimalTelemetryDeviceFile extends IAnimalTelemetryDevice {
+            sx={{
+              mt: -1,
+              mb: 3
+            }}>{`Vectronic KeyX File (Optional)`}</Typography>
           <TelemetryFileUpload attachmentType={AttachmentType.KEYX} index={props.index} />
         </>
       )}
       {props.deviceMake === 'Lotek' && (
         <>
-          <Typography sx={{ ml: 1, mb: 3 }}>{`Upload Lotek Config File`}</Typography>
+          <Typography
+            variant="body1"
+            color="textSecondary"
+            sx={{
+              mt: -1,
+              mb: 3
+            }}>{`Lotek Config File (Optional)`}</Typography>
           <TelemetryFileUpload attachmentType={AttachmentType.OTHER} index={props.index} />
         </>
       )}
-    </Paper>
+    </>
   );
 };
 
 const DeploymentFormSection = ({
   index,
-  deployments
+  deployments,
+  mode,
+  removeAction
 }: {
   index: number;
   deployments: IDeploymentTimespan[];
+  mode: TELEMETRY_DEVICE_FORM_MODE;
+  removeAction: (deployment_id: string) => void;
 }): JSX.Element => {
+  const [openDeleteDialog, setOpenDeleteDialog] = useState(false);
+  const [deploymentToDelete, setDeploymentToDelete] = useState('');
   return (
     <>
-      <Grid container spacing={2}>
+      <YesNoDialog
+        dialogTitle={'Remove deployment?'}
+        dialogText={`Are you sure you want to remove this deployment? 
+          If you would like to end a deployment / unattach a device, you should instead set the attachment end date. 
+          Please confirm that you wish to permanently erase this deployment.`}
+        open={openDeleteDialog}
+        yesButtonLabel="Delete"
+        noButtonLabel="Cancel"
+        yesButtonProps={{ color: 'error' }}
+        onClose={() => setOpenDeleteDialog(false)}
+        onNo={() => setOpenDeleteDialog(false)}
+        onYes={async () => {
+          await removeAction(String(deploymentToDelete));
+          setOpenDeleteDialog(false);
+        }}></YesNoDialog>
+      <Grid container spacing={3}>
         {deployments.map((deploy, i) => {
           return (
             <Fragment key={`deployment-item-${deploy.deployment_id}`}>
-              <Grid item xs={6}>
+              <Grid item xs={mode === TELEMETRY_DEVICE_FORM_MODE.ADD ? 6 : 5.5}>
                 <SingleDateField
                   name={`${index}.deployments.${i}.attachment_start`}
                   required={true}
-                  label={'Attachment Start'}
+                  label={'Start Date'}
                 />
               </Grid>
-              <Grid item xs={6}>
-                <SingleDateField name={`${index}.deployments.${i}.attachment_end`} label={'Attachment End'} />
+              <Grid item xs={mode === TELEMETRY_DEVICE_FORM_MODE.ADD ? 6 : 5.5}>
+                <SingleDateField name={`${index}.deployments.${i}.attachment_end`} label={'End Date'} />
               </Grid>
+              {mode === TELEMETRY_DEVICE_FORM_MODE.EDIT && (
+                <Grid item xs={1}>
+                  <IconButton
```

Comment:
nit: I think it looks a bit off, where the deployments are no longer aligned with the grid above due to this button. That being said, I don't really see a better option.